hack/multus/kubernetes-multus.yaml

```diff
@@ -1,25 +0,0 @@
----
-kind: ConfigMap
```

Comment:
why are we dropping hack/multus yamls?

Reply:
Multus shipped by CNAO, this is just a remnant of the past.